### PR TITLE
Website Design

### PR DESCRIPTION
**Pull request by EllaCodes4**
Contributing Initial Website Design/Front End Development 

- Converted website design in Figma into code 
- Contributing initial navbar, hero, about, maintainers, and footer components (Members section forthcoming)!

### CODE REVIEW

**Comment by FrancesCoronel** [comment]
@EllaCodes4 Thanks for the request. Make sure to also fill out your PR details with a summary of what you did

Screenshot 2023-03-04 at 10 04 53 AM

**Comment by FrancesCoronel** [comment]
@EllaCodes4 Let's remove the navbar links for now and then add those back in later once we figure out the anchor links issue. I also added a few more comments in the Vercel link. Once those are good, let's merge this in.

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "typescript.tsdk": "node_modules/typescript/lib",
+  "typescript.tsdk": "node_modules\\typescript\\lib",
```
Comment (FrancesCoronel) [inline .vscode/settings.json:2]: we can probably get rid of this change

File: README.md [hunk 2]
```diff
@@ -16,7 +16,12 @@ If you have feedback or are interested in contributing, feel free to create a ne
 
 ### Getting Started
 
-First, run the development server:
+If this is your very first time contributing, run the development server:
```
Comment (FrancesCoronel) [inline README.md:19]: instead of running development server, this is installing the dependencies

**Review by FrancesCoronel — CHANGES_REQUESTED** [review]
Looks awesome!! 💯 

If you could just address the two comments I made in the code and then a few comments I made on the visual preview, this should be good to go and we can merge this in!

File: .vscode/settings.json (hunk 1 above)
Comment (EllaCodes4) [reply .vscode/settings.json]: @FrancesCoronel --For some reason, this same change keeps coming up every time I go to commit! I've went ahead and discarded it though!

**Review by FrancesCoronel — APPROVED** [review]
Looks amazing! 💯

**Comment by EllaCodes4** [comment]
@FrancesCoronel Yay! How exciting!!!!

**Pull request merged**